webapp/app/utils/l2-cancellation-states.ts

```diff
@@ -8,3 +8,12 @@ export enum L2_CANCELATION_STATES {
   Expired = 6,
   Executed = 7,
 }
+
+export const VALID_CANCELLATION_STATES = [
+  L2_CANCELLATION_STATES.Active,
+  L2_CANCELLATION_STATES.Pending,
+];
+
+export function isValidCancellationState(state: L2_CANCELLATION_STATES | undefined): boolean {
```

Comment:
Throughout my reviews I've found myself looking at this function implementation everytime it came up, as I couldn't realize what it was doing just by the name (and couldn't remember thanks to my dumb memory). Perhaps we could change the naming to something like `isL2ProposalActive` or similar, as the state has more to do with the proposal rather than the cancellation.

Reply:
We agreed on leaving this for next release.